projects/laji-ui/src/lib/modal/modal/modal.component.scss

```diff
@@ -74,14 +86,26 @@
 @media (min-width: 992px) {
 	.lu-modal-lg {
     .lu-modal-content {
-			width: 900px;
+			width: $lg-width;
+    }
+    > .lu-modal-close-button {
+      left: calc(50% + $lg-width / 2 + $padding);
     }
-      > .lu-modal-close-button {
-        left: calc(50% + 455px);
-      }
 	}
 }
 
+
+@media (min-width: 1660px) {
```

Comment:
use the breakpoint variable here? `$xl-width`

Reply:
Right, sorry. Anyways these sort of feel like magic values when they should be related to breakpoints. Could maybe do something like `calc(map-get($breakpoints, "xl") + 60px)` (not sure if that syntax works but you get the idea)

Reply:
Not sure why the breakpoints should have exceptions here though

Reply:
Hmm... The breakpoints are weird numbers like 576, 768, 992, 1660. There's no pattern.

Modal widths are numbers like 300, 600, 900, 1600. There's a pattern (ok there's an exception but it's kinda related to the pattern still).

I don't see a pattern between the breakpoints and the widths. If we'd change some of the breakpoint values (I don't think we ever will), the width pattern would break.

Also I don't think they should be relative anyways - think about the small width. If the small breakpoint is changed, the width of the small modal would change for all widths and that's not necessarily desired.

Reply:
I probably should have read the commit more carefully... But it just feels weird to have exceptions in media queries. Like the entire purpose of the breakpoints is media queries like this :D I'm not sure if it actually matters though (does it matter that things wrap etc at different breakpoints? idk). I think the pr can be approved as is.